src/schematools/loaders.py

```diff
@@ -0,0 +1,35 @@
+import importlib
+import os
+from typing import TYPE_CHECKING, Callable, Optional
+
+from schematools.types import DatasetSchema
+from schematools.utils import dataset_schema_from_file, dataset_schema_from_url
+
+
+def get_loader(reference: str) -> Callable:
+    """Gets a loader callable from a string reference."""
+    module_name, separator, callable_name = reference.partition(":")
+    if not separator:
+        raise ValueError("Provide a fully qualified reference `package.module:callable`")
+    obj = importlib.import_module(module_name)
+    for attr in callable_name.split("."):
+        obj = getattr(obj, attr)
+    return obj
```

Comment:
This looks like a security breach waiting to happen.

Reply:
I think you're right, because this is used in the context of github actions. We do not know enough about that environment to be sure that no one touches the environment vars.